package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "grunt-node-inspector",
-  "version": "0.2.0",
+  "version": "0.2.1",
```

Comment:
Could you remove this diff and rebase to a single commit. Then I will merge and bump the version myself.